public/js/layout.js

```diff
@@ -1,6 +1,7 @@
 // Layout.js - all subcircuit layout related code is here
 
 // Function to toggle between layoutMode and normal Mode
+var allLayouts = {}; // Stores all layouts of SubCircuit
```

Comment:
It looks like we don't need this to fix this bug.
Just remove Scope.prototype.fixLayout() and its call from switchCircuit().

Reply:
Yes just removing the function call to `globalScope.fixLayout();` will fix this.

Reply:
It would be better to remove just one line instead of adding more lines.

Thanks, @satu0king and @fpintos